### ISSUE TITLE
Could not pass event PlayerMoveEvent to SimpleScore v3.12.2

### ISSUE THREAD

**Issue by rishabhk571** [r4g3baby/SimpleScore#59]
```
[05:11:01 ERROR]: Could not pass event PlayerMoveEvent to SimpleScore v3.12.2
java.lang.RuntimeException: An internal error occured.
        at com.comphenix.protocol.reflect.accessors.DefaultConstrutorAccessor.invoke(DefaultConstrutorAccessor.java:22) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.injector.StructureCache.lambda$newPacket$0(StructureCache.java:75) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.injector.StructureCache.newPacket(StructureCache.java:83) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.injector.StructureCache.newPacket(StructureCache.java:95) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.events.PacketContainer.<init>(PacketContainer.java:130) ~[ProtocolLib.jar:?]
        at com.r4g3baby.simplescore.scoreboard.handlers.ProtocolScoreboard.createScoreboard$lambda-0(ProtocolScoreboard.kt:27) ~[SimpleScore-3.12.2.jar:?]
        at java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1708) ~[?:?]
        at com.r4g3baby.simplescore.scoreboard.handlers.ProtocolScoreboard.createScoreboard(ProtocolScoreboard.kt:26) ~[SimpleScore-3.12.2.jar:?]
        at com.r4g3baby.simplescore.scoreboard.ScoreboardManager.updateScoreboardState$SimpleScore(ScoreboardManager.kt:89) ~[SimpleScore-3.12.2.jar:?]
        at com.r4g3baby.simplescore.scoreboard.listeners.PlayerListener.onPlayerMove(PlayerListener.kt:43) ~[SimpleScore-3.12.2.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor81.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:git-Paper-448]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.19.3.jar:git-Paper-448]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[paper-1.19.3.jar:git-Paper-448]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleMovePlayer(ServerGamePacketListenerImpl.java:1593) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundMovePlayerPacket.handle(ServerboundMovePlayerPacket.java:29) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundMovePlayerPacket$Pos.handle(ServerboundMovePlayerPacket.java:66) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:51) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1341) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1318) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1311) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1289) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1177) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[paper-1.19.3.jar:git-Paper-448]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: io.netty.handler.codec.DecoderException: Received unexpected null component
        at net.minecraft.network.FriendlyByteBuf.readComponent(FriendlyByteBuf.java:494) ~[?:?]
        at net.minecraft.network.protocol.game.ClientboundSetObjectivePacket.<init>(ClientboundSetObjectivePacket.java:33) ~[?:?]
        at jdk.internal.reflect.GeneratedConstructorAccessor76.newInstance(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
        at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
        at com.comphenix.protocol.reflect.accessors.DefaultConstrutorAccessor.invoke(DefaultConstrutorAccessor.java:16) ~[ProtocolLib.jar:?]
        ... 33 more
```

**Comment by rishabhk571** [comment]
```
[05:27:09 INFO]: UUID of player HIDANUKILLER is b005e42b-7b6b-3d3c-8345-29a8752a23bf
[05:27:09 ERROR]: Could not pass event PlayerJoinEvent to SimpleScore v3.12.2
java.lang.RuntimeException: An internal error occured.
        at com.comphenix.protocol.reflect.accessors.DefaultConstrutorAccessor.invoke(DefaultConstrutorAccessor.java:22) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.injector.StructureCache.lambda$newPacket$0(StructureCache.java:75) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.injector.StructureCache.newPacket(StructureCache.java:83) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.injector.StructureCache.newPacket(StructureCache.java:95) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.events.PacketContainer.<init>(PacketContainer.java:130) ~[ProtocolLib.jar:?]
        at com.r4g3baby.simplescore.scoreboard.handlers.ProtocolScoreboard.createScoreboard$lambda-0(ProtocolScoreboard.kt:27) ~[SimpleScore-3.12.2.jar:?]
        at java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1708) ~[?:?]
        at com.r4g3baby.simplescore.scoreboard.handlers.ProtocolScoreboard.createScoreboard(ProtocolScoreboard.kt:26) ~[SimpleScore-3.12.2.jar:?]
        at com.r4g3baby.simplescore.scoreboard.ScoreboardManager.updateScoreboardState$SimpleScore(ScoreboardManager.kt:89) ~[SimpleScore-3.12.2.jar:?]
        at com.r4g3baby.simplescore.scoreboard.ScoreboardManager.updateScoreboardState$SimpleScore$default(ScoreboardManager.kt:86) ~[SimpleScore-3.12.2.jar:?]
        at com.r4g3baby.simplescore.scoreboard.listeners.PlayerListener.onPlayerJoin(PlayerListener.kt:18) ~[SimpleScore-3.12.2.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor82.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:git-Paper-448]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.19.3.jar:git-Paper-448]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[paper-1.19.3.jar:git-Paper-448]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.players.PlayerList.placeNewPlayer(PlayerList.java:319) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.network.ServerLoginPacketListenerImpl.placeNewPlayer(ServerLoginPacketListenerImpl.java:202) ~[?:?]
        at net.minecraft.server.network.ServerLoginPacketListenerImpl.handleAcceptedLogin(ServerLoginPacketListenerImpl.java:183) ~[?:?]
        at net.minecraft.server.network.ServerLoginPacketListenerImpl.tick(ServerLoginPacketListenerImpl.java:85) ~[?:?]
        at net.minecraft.network.Connection.tick(Connection.java:585) ~[?:?]
        at net.minecraft.server.network.ServerConnectionListener.tick(ServerConnectionListener.java:233) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1565) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:447) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1397) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1173) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[paper-1.19.3.jar:git-Paper-448]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: io.netty.handler.codec.DecoderException: Received unexpected null component
        at net.minecraft.network.FriendlyByteBuf.readComponent(FriendlyByteBuf.java:494) ~[?:?]
        at net.minecraft.network.protocol.game.ClientboundSetObjectivePacket.<init>(ClientboundSetObjectivePacket.java:33) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
        at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
        at com.comphenix.protocol.reflect.accessors.DefaultConstrutorAccessor.invoke(DefaultConstrutorAccessor.java:16) ~[ProtocolLib.jar:?]
        ... 29 more
[05:27:09 INFO]: HIDANUKILLER joined the game
[05:27:09 INFO]: HIDANUKILLER[/117.215.160.67:60342] logged in with entity id 251 at ([world]-160.57979998419827, 63.0, -1005.713814629775)
[05:27:09 INFO]: [AuthMe] HIDANUKILLER logged in 117.215.160.67
[05:27:14 WARN]: [ProtocolLib] [DetailedErrorReporter] Internal exception count: 128!
[05:27:20 INFO]: [Not Secure] [Piercers] <HIDANUKILLER> oof
[05:27:20 ERROR]: Could not pass event PlayerMoveEvent to SimpleScore v3.12.2
java.lang.RuntimeException: An internal error occured.
        at com.comphenix.protocol.reflect.accessors.DefaultConstrutorAccessor.invoke(DefaultConstrutorAccessor.java:22) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.injector.StructureCache.lambda$newPacket$0(StructureCache.java:75) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.injector.StructureCache.newPacket(StructureCache.java:83) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.injector.StructureCache.newPacket(StructureCache.java:95) ~[ProtocolLib.jar:?]
        at com.comphenix.protocol.events.PacketContainer.<init>(PacketContainer.java:130) ~[ProtocolLib.jar:?]
        at com.r4g3baby.simplescore.scoreboard.handlers.ProtocolScoreboard.createScoreboard$lambda-0(ProtocolScoreboard.kt:27) ~[SimpleScore-3.12.2.jar:?]
        at java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1708) ~[?:?]
        at com.r4g3baby.simplescore.scoreboard.handlers.ProtocolScoreboard.createScoreboard(ProtocolScoreboard.kt:26) ~[SimpleScore-3.12.2.jar:?]
        at com.r4g3baby.simplescore.scoreboard.ScoreboardManager.updateScoreboardState$SimpleScore(ScoreboardManager.kt:89) ~[SimpleScore-3.12.2.jar:?]
        at com.r4g3baby.simplescore.scoreboard.listeners.PlayerListener.onPlayerMove(PlayerListener.kt:43) ~[SimpleScore-3.12.2.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor77.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:git-Paper-448]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.19.3.jar:git-Paper-448]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[paper-1.19.3.jar:git-Paper-448]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleMovePlayer(ServerGamePacketListenerImpl.java:1593) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundMovePlayerPacket.handle(ServerboundMovePlayerPacket.java:29) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundMovePlayerPacket$Pos.handle(ServerboundMovePlayerPacket.java:66) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:51) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1341) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1318) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1311) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1289) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1177) ~[paper-1.19.3.jar:git-Paper-448]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[paper-1.19.3.jar:git-Paper-448]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: io.netty.handler.codec.DecoderException: Received unexpected null component
        at net.minecraft.network.FriendlyByteBuf.readComponent(FriendlyByteBuf.java:494) ~[?:?]
        at net.minecraft.network.protocol.game.ClientboundSetObjectivePacket.<init>(ClientboundSetObjectivePacket.java:33) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
        at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
        at com.comphenix.protocol.reflect.accessors.DefaultConstrutorAccessor.invoke(DefaultConstrutorAccessor.java:16) ~[ProtocolLib.jar:?]
        ... 33 more
```

**Comment by r4g3baby** [comment]
Hey, what version of ProtocolLib are you using?

**Comment by rishabhk571** [comment]
> Hey, what version of ProtocolLib are you using?

Solved, just needed to update to 1.19.3 dev version of Protocol Lib, they haven't released stable version for 1.19.3 yet, that's why.

Thanks, you can close this issue now.
